web/src/components/Plugin/PluginDetail.tsx

```diff
@@ -190,49 +188,53 @@ const PluginDetail: React.FC<Props> = ({
       });
     });
   };
-  const handleModeChange = (value: PluginComponent.CodeMirrorMode) => {
+
+  const editorWillMount = (monaco: typeof monacoEditor) => {
+    fetchSchema(name, schemaType).then((schema)=> {
+      const schemaConfig = {
+        validate: true,
+        schemas: [
+          {
+            uri: `https://apisix.apache.org/`,
```

Comment:
May I know what's this opinion does?

Reply:
This uri seems like useless if the schema is already set, but this is required by type and must be a valid uri. So I just use `https://apisix.apache.org` as a placeholder.